compiler/integration-tests/package.json

```diff
@@ -7,19 +7,20 @@
     "build": "echo Integration Test build step",
     "test": "yarn test:browser",
     "test:browser": "web-test-runner",
-    "test:integration:browser": "web-test-runner test/integration/browser/**/*.test.ts",
-    "test:integration:browser:watch": "web-test-runner test/integration/browser/**/*.test.ts --watch",
+    "test:integration:browser": "web-test-runner test/integration/browser/recursion.test.ts",
+    "test:integration:browser:watch": "web-test-runner test/integration/browser/recursion.test.ts --watch",
```

Comment:
Need to revert this change before merging so we still run the old tests.

Reply:
Resolved in 82d5ba70c5d18ac9f7fd1c1fd0dc4c1f9c978aaa